### PR TITLE
Meta: Remove maintainers section

### PR DESCRIPTION
**Pull request by bfred-it**
As discussed in #923 

cc @sindresorhus @jgierer12 @hkdobrev @busches @paulmolluzzo

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ While this project is highly opinionated, this doesn't necessarily limit you fro
 
 ## Contribute
 
-See the [contribution guide](contributing.md).
+See the [contribution guide](contributing.md) and appear on the [contributors list](https://github.com/sindresorhus/refined-github/graphs/contributors)!
```
Comment (hkdobrev) [inline readme.md]: I'd prefer a bit different wording. Probably something like "and join the contributors".

**Comment by hkdobrev** [comment]
I'm up for removing the maintainers section, however, it's important to note that it serves a different purpose. It shows which are maintainers and have push access and not just contributors. GitHub differentiates these with comment badges - "Contributor" vs "Collaborator" (and also "Owner"). Sometimes when you're posting a new issue/PR and even more so when you don't have a response for some time, you'd want to ping maintainers and not just contributors. I usually find them on other repos by finding who has merged PRs lately. If there's such a section in the readme, I'd usually use that.

These are just some thoughts which could result in a new feature for RG - probably marking collaborators in the graph based on API calls, `MAINTAINERS` or `.mailmap` files in the repo.

**Comment by paulmolluzzo** [comment]
> These are just some thoughts which could result in a new feature for RG - probably marking collaborators in the graph based on API calls, MAINTAINERS or .mailmap files in the repo.

This is a great idea @hkdobrev!

**Comment by bfred-it** [comment]
Isn’t it kind of obvious that the maintainers are those who have more than a couple commits on it?

**Comment by jgierer12** [comment]
I don't think we need the maintainers section either; as you said you can usually guess by looking at the contributions or by looking at recently merged PRs (I usually do that anyway to get an idea of the project's conventions).

What I meant by the 'important stuff' (https://github.com/sindresorhus/refined-github/pull/923#issuecomment-354427970) is information about using the extension (i.e., the *Install*, *Customization*, *Contribute*, and maybe *Related* sections). That's what the readme is for, IMO. I don't like it when libraries put their whole API documentation in the readme, and the *Features* section feels just like that. But I don't feel strongly about it so I'm fine with leaving it as it is.

**Comment by bfred-it** [comment]
> These are just some thoughts which could result in a new feature for RG - probably marking collaborators in the graph based on API calls, MAINTAINERS or .mailmap files in the repo.

Marking owners/collaborators in the Contributors page sounds good, if that's what you're thinking, however those two files don't have a fixed structure and may be impossible to link them to their GitHub user reliably. Perhaps open a feature request if it can be done via API.

**Pull request merged**